addons/compat_rhs_afrf3/compat_rhs_afrf3_repair/CfgEventHandlers.hpp

```diff
@@ -0,0 +1,5 @@
+class Extended_PreInit_EventHandlers {
+    class ADDON {
+        init = QUOTE(call COMPILE_SCRIPT(XEH_preInit));
```

Comment:
This might also work? 🤢 

```suggestion
    class SUBADDON {
        init = QUOTE(call COMPILE_SCRIPT(SUBADDON\XEH_preInit));
```

Reply:
Nah, I'll revert the change.